model_analyzer/config/run/model_run_config.py

```diff
@@ -96,7 +97,13 @@ def representation(self):
         Returns a representation string for the ModelRunConfig that can be used
         as a key to uniquely identify it
         """
-        return self.perf_config().representation()
+        repr_string = self.perf_config().representation()
```

Comment:
Should we just change this in PerfConfig instead?

Reply:
I didn't feel like it belonged in PerfConfig because I don't think that class should be mucking w/ the representation. It should just be returning it unmutated (what if another class wanted repr fields that we chose to remove because of how we used it in ModelRunConfig.

Reply:
Yeah it all depends on how we expect it to be used. This is the only place that function code is used, and longer term I would like it to be even more stripped down. In my opinion it should have the minimum information needed to uniquely differentiate it from other runs.

Reply:
I've moved the logic into perf_config.py